category/execution/ethereum/execute_transaction_test.cpp

```diff
@@ -210,12 +318,12 @@ TEST(TransactionProcessor, refunds_delete)
 
     // Sets s[0] = 1 if passed any data, clears s[0] if data is empty.
     auto const contract_code =
-        evmc::from_hex("0x3615600a5760015f55005b5f5f55").value();
+        evmc::from_hex("0x3615600b576001600055005b6000600055").value();
```

Comment:
modified code to work with all evm revisions by replacing all `0x5f` (`PUSH0`) opcodes with `0x6000` (`PUSH1 0`) in both tests.